com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/Log.java

```diff
@@ -0,0 +1,114 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package com.microsoft.java.debug.core;
+
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This helper class provides static methods for a default instance of <code>java.util.logging.Logger</code>,a simple code like
+ * <code>Log.info("Sample trace: %d", 10)</code> is able to replace <code>logger.info(String.format("Sample trace: %d", 10))</code>.
+ * Using a very short name to better code fitness.
+ */
+public final class Log {
```

Comment:
Why do we need to wrap around the JDK logger? I didn't see much value added. Am I missing anything?

Reply:
This PR is mainly to focus on simplifying the logging code to better code.

Reply:
Trace user data is also include in Log.java, Log will be the only class need to record logs and user datas.